compiler/rustc_codegen_cranelift/src/unsize.rs

```diff
@@ -58,24 +86,29 @@ fn unsize_thin_ptr<'tcx>(
         (&ty::Adt(def_a, _), &ty::Adt(def_b, _)) => {
```

Comment:
Above `unsized_info` is getting `None` passed as `old_info` twice. I think these need to be passed the real `old_info`.

Reply:
Currently typechecking will ensure `old_info` is `None` in these cases. But it's no harm and more consistent to always pass `old_info` inside. I'll do this.

Reply:
Done.